docs/starters.yml

```diff
@@ -1061,3 +1061,26 @@
     - Antd component library pre-installed
     - Uses gatsby-plugin-antd for modular imports
     - Customize the theme of Antd with `modifyVars`
+- url: https://gatsby-starter-typescript-power-blog.majidhajian.com/
+  repo: https://github.com/mhadaily/gatsby-starter-typescript-power-blog
+  description: Minimal Personal Blog with Gatsby and Typescript
+  tags:
+    - PWA
+    - Blog
+    - Personal website
+    - TypeScript
+    - Progressive web app
```

Comment:
Please omit this tag as you're already using `PWA`

Reply:
done